src/growl.c

```diff
@@ -466,13 +473,17 @@ int growl_tcp_notify(const char *const server,
             goto leave;
         } else {
             int len = strlen(line);
-            /*fprintf(stderr, "%s\n", line);*/
-            if (strncmp(line, "GNTP/1.0 -ERROR", 15) == 0) {
-                if (strncmp(line + 15, " NONE", 5) != 0) {
-                    fprintf(stderr, "failed to post notification\n");
+            //No need to use strncmp, as ERRSTR is well known.
+            if (!strcmp(line, ERRSTR)) {
```

Comment:
Thinking about it a bit more you might have a point if the client does not abide to the protocol. I'll fix it.